hedera-node/hedera-app/src/test/resources/initcode/EmitBlockTimestamp.bin

```diff
@@ -0,0 +1 @@
+608060405234801561001057600080fd5b506104be806100206000396000f3fe608060405234801561001057600080fd5b506004361061004c5760003560e01c806327e86d6e146100515780632e4c1d021461006f578063672829a614610079578063e7dc4e2214610098575b600080fd5b6100596100b6565b6040516100669190610227565b60405180910390f35b6100776100cb565b005b61008161014a565b60405161008f92919061025b565b60405180910390f35b6100a0610164565b6040516100ad9190610342565b60405180910390f35b60006001436100c59190610393565b40905090565b7fdb7bcd08f1cb9a081d4a97aa8df2caca5caf3948df72cf63559bd93e2f44ffed426040516100fa91906103c7565b60405180910390a160004390506000814090507f4977e6b2d0d386b4fd3201b04b268b23d9e2ddcf6cab18a51ce2cc2978adc131828260405161013e92919061025b565b60405180910390a15050565b60008060014361015a9190610393565b9150814090509091565b6060600060ff9050600061010067ffffffffffffffff81111561018a576101896103e2565b5b6040519080825280602002602001820160405280156101b85781602001602082028036833780820191505090505b50905060005b828110156102055780436101d29190610393565b408282815181106101e6576101e5610411565b5b60200260200101818152505080806101fd90610440565b9150506101be565b50809250505090565b6000819050919050565b6102218161020e565b82525050565b600060208201905061023c6000830184610218565b92915050565b6000819050919050565b61025581610242565b82525050565b6000604082019050610270600083018561024c565b61027d6020830184610218565b9392505050565b600081519050919050565b600082825260208201905092915050565b6000819050602082019050919050565b6102b98161020e565b82525050565b60006102cb83836102b0565b60208301905092915050565b6000602082019050919050565b60006102ef82610284565b6102f9818561028f565b9350610304836102a0565b8060005b8381101561033557815161031c88826102bf565b9750610327836102d7565b925050600181019050610308565b5085935050505092915050565b6000602082019050818103600083015261035c81846102e4565b905092915050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b600061039e82610242565b91506103a983610242565b92508282039050818111156103c1576103c0610364565b5b92915050565b60006020820190506103dc600083018461024c565b92915050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b600061044b82610242565b91507fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff820361047d5761047c610364565b5b60018201905091905056fea264697066735822122022024da7048e20c190d12620cf12562191616f7e8a0376df13f3f6b709e054ca64736f6c63430008120033
```

Comment:
How did the binary change without the `.sol` changing?  I.e., shouldn't the `.sol` be in this PR too?  (And the `.json` too if a method was added/changed.)